### DISCUSSION TITLE
Helper libs used inside Tye2

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#46] [Ideas]
I want to use fluent assertion (https://www.nuget.org/packages/FluentAssertions) and Refit (https://www.nuget.org/packages/Refit) inside Tye2, do you have something against? Both already used in #46

**Comment by razvangoga** [comment]
none in principle against any of them (i use them successfully in other projects), assuming there are no licencing issues - can you check this?

fluent assertions i assume is for the unit tests.
what would you use Refit for?

**Reply by drakon660** [reply]
#43 please check this You will see, basically is contract oriented http client wrapper. No license issues.
